site/docs/v1/tech/tutorials/integrate-react.md

```diff
@@ -0,0 +1,123 @@
+navcategory: getting-started
+prerequisites: nodejs
+technology: React
+language: JavaScript
+---
+
+## Integrate Your {{page.technology}} Application With FusionAuth
+
+{% include docs/integration/_intro.md %}
+
+## Prerequisites
+
+{% include docs/integration/_prerequisites.md %}
+
+## Download and Install FusionAuth
+
+{% include docs/integration/_install-fusionauth.md %}
+
+## Create a User and an API Key
+
+{% include docs/integration/_add-user.md %}
+
+## Configure FusionAuth
+
+Next, you need to set up FusionAuth. This can be done in different ways, but we’re going to use the Typescript client library. The below instructions use `npm` on the command line, but you can use the client library with an IDE of your preference as well.
+
+First, make a directory:
+
+```shell
+mkdir setup-fusionauth && cd setup-fusionauth
+```
+
+Now, copy and paste the following file into `package.json`.
+
+```javascript
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-client-libraries/main/typescript/package.json %}
+```
+
+Now you need to install the dependencies in `package.json`.
+
+```shell
+npm install
+```
+
+Then copy and paste the following file into `setup.js`. This file uses the [FusionAuth API](/docs/v1/tech/apis/) to configure an Application, CORS and more to allow for easy integration. 
+
+```javascript
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-client-libraries/main/typescript/setup.js %}
+```
+
+Then, you can run the setup script.
+
+```shell
+npm run setup
+```
+
+If you want, you can http://localhost:9011[login to your instance] and examine the new Application the script created for you.
+
+## Create Your {{page.technology}} Application
+
+Now you are going to create a {{page.technology}} application. While this section uses a simple {{page.technology}} application without any framework, you can use the same steps to integrate any {{page.technology}} application with FusionAuth.
+
+First, make a directory:
+
+```shell
+mkdir ../setup-react && cd ../setup-react
+```
+
+Next, create a simple {{page.technology}} template using vite. Using this lets you easily integrate FusionAuth. For a production application you'll probably use something like [NextJS or Remix](/docs/quickstarts/#single-page-app).
+
+```shell
+npm create vite@latest react-app -- --template react
+```
+
+Now, change into the `react-app` directory and install the needed packages.
+
+```shell
+cd react-app && npm install
+```
+
+You can start up the server and visit the URL displayed to ensure the default application works.
+
+```shell
+npm run dev
+```
+
+You'll want to open another terminal window to continue. Edit `src/App.jsx` to make changes to the view to test out the automatic reloading.
+
+{% include _callout-tip.liquid content="If you are using a different port for your {{page.technology}} app, update the redirect URL in the setup script, or modify it manually. It needs to match exactly." %}
```

Comment:
This is giving me an error when building.

> Liquid Exception: Invalid syntax for include tag: app, update the redirect URL in the setup script, or modify it manually. It needs to match exactly." Valid syntax: {% include file.ext param='value' param2='value' %} in docs/v1/tech/tutorials/integrate-react.md

I'm using Jekyll 4.3.2 and Liquid 4.0.4. I had to change it to this to make it work.

```asciidoc
{% capture content %}If you are using a different port for your {{page.technology}} app, update the redirect URL in the setup script, or modify it manually. It needs to match exactly.{% endcapture %}
{% include _callout-tip.liquid content=content %}
```

Reply:
thanks!